app/templates/eventView.html

```diff
@@ -51,7 +51,7 @@ <h3><br>Description</h3><p>{{eventData.description|urlize}}</p><br>
                         <p>This event is a recurring event.
                             {% if eventData.nextRecurringEvent %}
                             The next event in the series occurs on <b>{{eventData.nextRecurringEvent.startDate.strftime("%B %-d")}}</b>.
-                            <a href="/eventsList/{{eventData.nextRecurringEvent.id}}/view">Click here for more information.</a>
+                            <a href="/eventsList/{{eventData.nextRecurringEvent.id}}/view">Look at next event.</a>
```

Comment:
Instead of having another sentence to link to the next event make the date of the next event the link. 

So, on this page: 
![image](https://user-images.githubusercontent.com/26262045/186761157-b44ba561-cb26-44f2-b96a-a911d148e7cc.png)

September 1 would be the link to the next event in the series.